components/toolbars/ToolBars.vue

```diff
@@ -16,13 +33,18 @@
         :txDialogActionRoute="getTxDialogActionRoute"
         :txDialogActionText="getTxDialogActionText"
         @onClose="closeTxDialog" />
-      <loading-toolbar :isLoading="getIsLoading" :isInTransaction="getIsInTransaction"/>
+
+      <loading-toolbar
+        :isLoading="getIsLoading"
+        :isInTransaction="getIsInTransaction" />
+
       <tx-toolbar
         v-if="getPendingTx"
-        :txHash="getPendingTx"
-        :txInfo="getPendingTxInfo"
+        :txHash="'getPendingTx'"
+        :txInfo="'getPendingTxInfo'"
```

Comment:
'getPendingTxInfo' will become the string `getPendingTxInfo` instead of object?

Reply:
@williamchong007 Fixed